modules/core/src/main/scala/gem/Program.scala

```diff
@@ -1,8 +1,18 @@
 package gem
 
-case class Program(id: Program.Id, title: String)
+import scalaz._, Scalaz._
+
+case class Program[A](id: Program.Id, title: String, observations: List[A])
 
 object Program {
+
   type Id = edu.gemini.spModel.core.ProgramId
```

Comment:
Right, that was my concern, those clases are mostly Java so they'd break in scala.js. In ocs3 we have simple scala classes for those classes and we copy back and forth to the ocs core classes, but it would be good to avoid that